doc/nrf/releases/release-notes-latest.rst

```diff
@@ -209,8 +229,50 @@ The following list summarizes the most important changes inherited from upstream
       * ``LINE_CTRL_DCD``
       * ``LINE_CTRL_DSR``
 
+  * SPI:
+
+    * Added support for SPI emulators.
+
   * USB:
 
     * Fixed handling of zero-length packets (ZLP) in the Nordic Semiconductor USB Device Controller driver (usb_dc_nrfx).
     * Fixed initialization of the workqueue in the usb_dc_nrfx driver, to prevent fatal errors when the driver is reattached.
     * Fixed handling of the SUSPEND and RESUME events in the Bluetooth classes.
+
+* Build system:
+
+  * Ensured that shields can be placed in other BOARD_ROOT folders.
+  * Added basic support for Clang 10 with x86.
+
+* System:
+
+  * Added an API that provides a printf family of functions (for example, :c:func:`cbprintf`) with a callback on character output, to perform in-place streaming of the formatted string.
+  * Updated minimal libc to print stderr just like stdout.
+  * Added an ``abort()`` function to minimal libc.
+  * Updated the ring buffer to allow using the full buffer capacity instead of forcing an empty slot.
+  * Added a :c:macro:`CLAMP` macro.
+  * Added a feature for post-mortem analysis to the tracing library.
+
+* Samples:
+
+  * Added :ref:`zephyr:nrf-ieee802154-rpmsg-sample`.
+  * Added :ref:`zephyr:cloud-tagoio-http-post-sample`.
+  * Added :ref:`zephyr:sockets-civetweb-websocket-server-sample`.
+  * :ref:`zephyr:led_ws2812_sample`: Updated to force SPIM on nRF52 DK.
+  * :ref:`zephyr:cfb_custom_fonts`: Added support for ssd1306fb.
+  * :ref:`zephyr:gsm-modem-sample`: Added suspend/resume shell commands.
+
+* Logging:
+
+  * Added STP transport and raw data output support for systrace.
+
+* Modules:
+
+  * Introduced a :option:`CONFIG_MBEDTLS_MEMORY_DEBUG` option for mbedtls.
+  * Updated LVGL to v7.6.1.
+  * Updated libmetal and openamp to v2020.10.
+
+* Other:
+
+  * Added initial LoRaWAN support.
+  * Updated west to fail if a HEX file has UICR data and ``--erase`` was not specified.
```

Comment:
Is that what you were trying to say?

Reply:
The "nrfjprog" part is important. How about "Updated `west flash` support for `nrfjprog` to fail if the HEX file has UICR data and `--erase` was not specified." or so?

Reply:
OK, sounds good.